packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/Converters.kt

```diff
@@ -269,76 +271,75 @@ internal interface CoreIntConverter : StorageTypeConverter<Long> {
 }
 
 internal object ByteConverter : CoreIntConverter, CompositeConverter<Byte, Long>() {
-    override inline fun fromPublic(value: Byte?): Long? = byteToLong(value)
```

Comment:
Are these critical to remove. While introducing the converter infrastructure I inspected the generated byte code and it seemed to have quite significant impact on the number of instructions if the calls were inlined. Don't know if that is measurable though, so maybe not a issue. But I would just be more comfortable to leave them until verified that it doesn't have an impact.

Reply:
The warning is that these inlines only work well if called directly and not via the interface they are implementing. 

Now that you point to it I found some places where they are invoked directly. Im restoring them and supressing the warning.

Reply:
They are also called directly from the generated IR code. Or at least was when I implemented it in the first place.